frontend/src/ResourceTreeNode.svelte

```diff
@@ -76,6 +76,18 @@
     color: var(--main-bg-color);
   }
 
+  .modified {
+    text-decoration-line: underline;
+    text-decoration-color: #dc4e47;
```

Comment:
We want to avoid hard-coding CSS colors inside components if possible. This makes them less portable, and means we can't change the colors programmatically. This should probably be a CSS variable set at the root in `global.css`.